test/stdlib/DictionaryCompactMapValues.swift

```diff
@@ -13,12 +13,8 @@ tests.test("DefaultReturnType") {
 }
 
 tests.test("ExplicitTypeContext") {
-  expectEqual(["a":"1","c":"3"],
-    ["a":"1","b":nil,"c":"3"].compactMapValues({$0})
-  )
-  expectEqual(["a": 1, "b": 2],
-    ["a":"1","b":"2", "c":"three"].compactMapValues(Int.init)
-  )
+  expectEqual(["a":"1","c":"3"], ["a":"1","b":nil,"c":"3"].compactMapValues({$0}))
+  expectEqual(["a": 1, "b": 2], ["a":"1","b":"2", "c":"three"].compactMapValues(Int.init))
```

Comment:
Sorry, I didn't mean to ask you to remove newlines here; these should be wrapped to 80 characters per line. I meant that there was (and still is) inconsistent use of spaces after colons and commas.

Reply:
@xwu  I misunderstood what you said.  now fix it!